packages/gamut-labs/src/GlobalFooter/FooterNavLinks/CompanyLinks.tsx

```diff
@@ -218,33 +223,18 @@ export const CompanyLinks: React.FC<CompanyLinksProps> = ({
     <Box>
       <FooterHeading>Resources</FooterHeading>
       <FooterLinkItems>
-        <FooterLinkItem>
-          <Anchor
-            href="https://codecademy.com/resources/blog/"
-            onClick={(event) => onClick({ event, target: 'blog' })}
-            variant="interface"
-          >
-            Blog
-          </Anchor>
-        </FooterLinkItem>
-        <FooterLinkItem>
-          <Anchor
-            href="/resources/cheatsheets/all"
-            onClick={(event) => onClick({ event, target: 'cheatsheets_home' })}
+        {resourcesList.map((resource) => (
+          <FooterLinkItemWithAnchor
+            key={resource.id}
+            footerOnClick={onClick}
+            trackingTarget={resource.trackingTarget}
+            href={resource.href}
             variant="interface"
+            target={resource.newTab ? 'blank' : ''}
```

Comment:
nit: You may want to consider a destructuring to read the markdown easier:

```suggestion
        {resourcesList.map(({ id, trackingTarget, href, newTab }) => (
          <FooterLinkItemWithAnchor
            key={id}
            footerOnClick={onClick}
            trackingTarget={trackingTarget}
            href={href}
            variant="interface"
            target={newTab ? 'blank' : ''}
```

Reply:
word!